build/builder.sh

```diff
@@ -120,10 +120,15 @@ vols="${vols} --volume=${HOME}/.yarn-cache:${container_home}/.yarn-cache${cached
 
 # If we're running in an environment that's using git alternates, like TeamCity,
 # we must mount the path to the real git objects for git to work in the container.
-alternates_file=${cockroach_toplevel}/.git/objects/info/alternates
-if test -e "${alternates_file}"; then
-  alternates_path=$(cat "${alternates_file}")
-  vols="${vols} --volume=${alternates_path}:${alternates_path}${cached_volume_mode}"
+# alternates_file=${cockroach_toplevel}/.git/objects/info/alternates
```

Comment:
Wait why are you including this change in this PR? It seems unrelated.

Reply:
It's necessary due to something that changed in teamcity. We don't want to pull in any more than we have to on top of v20.1.0-rc.2, which means we need to duplicate that commit on this PR branch.